File: engine/model/src/main/scala/tools/packet/character/interaction/MerchantVisitorLeave.scala
package tools.packet.character.interaction

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class MerchantVisitorLeave(private var _slot: Int) extends PacketInput {
  def slot: Int = _slot

  override def opcode(): SendOpcode = SendOpcode.PLAYER_INTERACTION
}

File: engine/engine-model/src/main/java/tools/packet/character/interaction/MerchantVisitorLeave.java
package tools.packet.character.interaction;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record MerchantVisitorLeave(Integer slot) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.PLAYER_INTERACTION;
   }
}